sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationAsyncClientTest.java

```diff
@@ -1217,7 +1221,7 @@ public void listRevisionsWithPaginationAndRepeatIterator(HttpClient httpClient,
     @MethodSource("com.azure.data.appconfiguration.TestHelper#getTestParameters")
     public void listConfigurationSettingsWithPagination(HttpClient httpClient, ConfigurationServiceVersion serviceVersion) {
         client = getConfigurationAsyncClient(httpClient, serviceVersion);
-        final int numberExpected = 50;
+        final int numberExpected = 13;
```

Comment:
Any reason to reduce the number? 50 would definitely will have second page of response, I remember the number around 28 ish

Reply:
That is what the updated record returned and it passes for this number. Any reason not to update this or is it always supposed to be 50?